src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,117 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.UserRegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static final String VALID_LOGIN = "abcdef";
+    private static final String VALID_PASSWORD = "123456";
+    private static final String LOGIN_LESS_THAN_MIN = "abcde";
+    private static final String PASSWORD_LESS_THAN_MIN = "12345";
+    private static final int AGE_LESS_THAN_MIN = 15;
+    private static RegistrationServiceImpl registrationService;
```

Comment:
```suggestion
    private static RegistrationService registrationService;
```
use abstract types for variable types